packages/patternfly-3/patternfly-react/package.json

```diff
@@ -63,10 +63,10 @@
     "build:babel:cjs": "cross-env BABEL_ENV=production:cjs babel src --out-dir dist/js",
     "build:babel:esm": "cross-env BABEL_ENV=production:esm babel src --out-dir dist/esm",
     "build:less": "shx mkdir -p dist/less && shx cp -r less/* dist/less",
-    "build:sass": "shx mkdir -p dist/sass && shx cp -r sass/patternfly-react/* dist/sass && cross-env node-sass --output-style compressed --include-path sass --include-path $npm_package_sassIncludes_patternfly --include-path $npm_package_sassIncludes_bootstrap --include-path $npm_package_sassIncludes_fontAwesome -o dist/css sass/patternfly-react.scss"
+    "build:sass": "shx mkdir -p dist/sass && shx cp -r sass/patternfly-react/* dist/sass && cross-env node-sass --output-style compressed --include-path sass --include-path $npm_package_sassIncludes_patternfly --include-path $npm_package_sassIncludes_bootstrap --include-path $npm_package_sassIncludes_fontAwesome --importer=build/node-sass-tilde-importer.js -o dist/css sass/patternfly-react.scss && cleancss -o dist/css/patternfly-react.css dist/css/patternfly-react.css"
```

Comment:
It fails for me locally, are there any new steps involved in running a build that I've maybe missed?

<img width="1183" alt="screen shot 2019-02-13 at 12 19 47 pm" src="https://user-images.githubusercontent.com/5942899/52730637-dc9fdc80-2f89-11e9-9ae4-02ac63058982.png">

Reply:
Hm, I might have installed `cleancss` globally and referenced that by mistake. Did you run `yarn install`? Maybe we need to use the direct path to cleancss in node_modules. Do you know how the `node-sass` call works here without being a path to node_modules?

Reply:
Yea I did a full rebuild with the repo. I haven't had a chance to dig deep into the changes, just got so far as running it and seeing the failure. You can try `git clean -xfd` to blow away all the generated files and directories, and maybe uninstall cleancss globalls, then run a clean install and try again? In case it still works for you and I've goofed this up, let me know and I'll be happy to try again.

Reply:
Will do. Thanks for taking a look.

Reply:
@seanforyou23 give it a try now. I think I had cleancss in the wrong package.json file. Monorepo problems